landingai/pipeline/frameset.py

```diff
@@ -141,13 +142,16 @@ def from_array(cls, array: np.ndarray, is_bgr: bool = True) -> "Frame":
         im = Image.fromarray(array)
         return cls(image=im)
 
-    def run_predict(self, predictor: Predictor) -> "Frame":
+    def run_predict(self, predictor: Predictor, reuse_session: bool = False) -> "Frame":
         """Run a cloud inference model
         Parameters
         ----------
         predictor: the model to be invoked.
+        reuse_session
+            Whether to reuse the session for sending multiple inference requests. By default, the session is reused to improve the performance. If you want to send multiple requests in parallel, set this to False.
```

Comment:
Can we rephrase this to `Whether to reuse the same HTTP session (...)`  and `If you are sending requests from multiple threads, set this to False`.